markdown_xblock/static/html/lms.html

```diff
@@ -1 +1,7 @@
-<div class="markdown_block">{{ self.html | safe }}</div>
+{% if self.classes %}
+<div class="{{ self.classes | join:' ' }}">
```

Comment:
Any particular reason for not setting the additional classes directly on the `markdown_xblock` div? As in:

```jinja2
<div class="markdown_block{% if self.classes %} {{ self.classes | join:' ' }}{% endif %}">
```

Reply:
Nope, just didn't think of it :)

Reply:
Also, this just occurred to me: shouldn't the class name be `markdown_xblock` rather than `markdown_block`?

Reply:
Sure, it probably should.